src/fetchtastic/download/android.py

```
@@ -0,0 +1,672 @@
+            List[Release]: Filtered list of prereleases
+        """
+        # Check if prereleases are enabled in config
+        check_prereleases = self.config.get(
+            "CHECK_APK_PRERELEASES", self.config.get("CHECK_PRERELEASES", False)
+        )
+
+        if not check_prereleases:
+            return []
+
+        version_manager = VersionManager()
+        prerelease_manager = PrereleaseHistoryManager()
```

Ruff (0.14.8)

Local variable `prerelease_manager` is assigned to but never used

Remove assignment to unused variable `prerelease_manager`

(F841)